_includes/sites.html

```diff
@@ -1,5 +1,7 @@
 {% capture methods %}{{ item.method }}{% endcapture %}
 
+{% if methods == 'colorscheme' %}
```

Comment:
This should be an elsif, otherwise it won't compile

Reply:
Thanks for pointing that out. I just fixed the missing elsif statement.